### ISSUE TITLE
Unnamed / Global Namespaces - generating strrpos offset error

### ISSUE THREAD

**Issue by Ashfitzg** [kahlan/kahlan#233]
When running Kahlan coverage reports on classes formatted as below:

```php
namespace {
   class foo {
   }
}
```
Kahlan generates the following error while trying to format names ready for report display.
``` 
Warning: strrpos(): Offset is greater than the length of haystack string in kahlan/src/Reporter/Coverage.php on line 278 
```

**Comment by jails** [comment]
Thanks for reporting :+1: , should be fixed in the [dev branch](https://github.com/kahlan/kahlan/commit/f3505a1987792eb9ab516a18d1f07a2e0b36dd64)

**Comment by jails** [comment]
It's now fixed in `3.1.11`
